crates/core/tedge_agent/src/agent.rs

```diff
@@ -198,16 +200,16 @@ impl SmAgent {
         info!("Starting tedge agent");
 
         let mut mqtt = Connection::new(&self.config.mqtt_config).await?;
+        let plugins_path = self.config.sm_home.join(SM_PLUGINS);
```

Comment:
Would be better if we rename `plugins_path` to `sm_plugins_path`. Since there could be other plugins as well right?

Reply:
sure